docs/kms/README.md

```diff
@@ -221,7 +221,7 @@ A KMS master key consists of a master-key ID (CMK) and the 256 bit master key en
 A KMS master key can be specified directly using:
 
 ```
-export MINIO_SSE_MASTER_KEY=my-minio-key:6368616e676520746869732070617373776f726420746f206120736563726574
+export MINIO_KMS_LOCAL_MASTER_KEY=my-minio-key:6368616e676520746869732070617373776f726420746f206120736563726574
```

Comment:
this variable needs to be made consistent in code & Readme - is MINIO_KMS_MASTER_KEY elsewhere.